### PR TITLE
Fix bundles landing accessibility

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?
We need the bundle landing page to as accessible as possible. Therefore, in this PR we fix the tab flow of the page and additionally we add the onKeyBehaviour for the links that are using onClick functions.

[**Trello Card**](https://trello.com/c/IyG06KgO/654-accessibility-issue-not-possible-to-navigate-properly-on-supporttheguardiancom-without-a-mouse)

### Contribute with the keyboard
contribute

### Tab flow
tabflow

### CODE REVIEW

File: assets/components/ctaLink/ctaLink.jsx [hunk 1]
```diff
@@ -12,14 +12,27 @@ type PropTypes = {
...
   onClick?: () => void,
+  tabIndex?: number,
 };
 
+// ----- Functions ----- //
+
+const onKeyPressHandler = (handler?: () => void): (event: Object) => void =>
+(event: Object) => {
+  // Check to see if space or enter were pressed
+  if (event.keyCode === 32 || event.keyCode === 13) {
```
Comment (davidfurey) [inline assets/components/ctaLink/ctaLink.jsx]: Might be clearer as:
```
const CarriageReturnCode = 13
const SpaceCode = 32
if ([CarriageReturnCode, SpaceCode].includes(event.keyCode)) {
  event.preventDefault();
  if (handler()) {
    handler();
  }
};
```

File: assets/components/ctaLink/ctaLink.jsx [hunk 2]
```diff
@@ -12,14 +12,27 @@ type PropTypes = {
   text: string,
   url?: string,
   onClick?: () => void,
+  tabIndex?: number,
 };
 
+// ----- Functions ----- //
+
+const onKeyPressHandler = (handler?: () => void): (event: Object) => void =>
```
Comment (davidfurey) [inline assets/components/ctaLink/ctaLink.jsx]: Could the method name be more informative, e.g.  `clickSubstitueKeyPressHandler` or something?

File: assets/components/ctaLink/ctaLink.jsx [hunk 3]
```diff
@@ -31,5 +44,6 @@ export default function CtaLink(props: PropTypes) {
 CtaLink.defaultProps = {
   url: null,
   onClick: null,
+  tabIndex: 0,
```
Comment (davidfurey) [inline assets/components/ctaLink/ctaLink.jsx]: Was this really all that was needed to fix the tab-indexing problem Mike raised?

Comment (svillafe) [inline assets/components/ctaLink/ctaLink.jsx]: It is a combination of things. Before the elements were not accessible via `tab` keystrokes. In order to achieve this, the elements that build up the page should be interactive web components. We had interactive components but due to design, they had the CSS property `display: none`, therefore they were not part of the page. We changed that in 5503c9d2a66976792400b6f69994509120cc0dc4. Additionally, when the links are defined via `onClick` function, the `space bar` and the `enter` keys do not achieve the desired behaviour (act as clicks). Consequently, we added that behaviour in 31aafc83be26d3b861399385bad7cf70e39b3eba.

**Comment by svillafe** [comment]
After an offline discussion with @michaelwmcnamara and @Ap0c, we decided to skip the prices placeholder and we discovered another problem with the `Patron`'s links.

**Pull request merged**